src/library/scala/util/Random.scala

```diff
@@ -43,11 +43,21 @@ class Random(val self: java.util.Random) extends AnyRef with Serializable {
    */
   def nextDouble(): Double = self.nextDouble()
 
+  /** Returns the next pseudorandom, uniformly distributed double value
+   *  between min and max from this random number generator's sequence.
+   */
+  def between(min: Double, max: Double): Double = min + self.nextDouble() % (max - min + 1.0)
+
   /** Returns the next pseudorandom, uniformly distributed float value
    *  between 0.0 and 1.0 from this random number generator's sequence.
    */
   def nextFloat(): Float = self.nextFloat()
 
+   /** Returns the next pseudorandom, uniformly distributed float value
+   *  between min and max from this random number generator's sequence.
+   */
+  def between(min: Float, max: Float): Float = min + self.nextFloat() % (max - min + 1.0f)
```

Comment:
The most confusing part for me when working with ranges is inclusive vs exclusive bounds. This should be made clear in every case (like this doc comment). I'd go a step further and add it to the parameter names (e.g. `minInclusive: Float, maxExclusive: Float`). You're not going to use these as named parameters anyway and the descriptive names will be shown by IDE auto-complete.